lib/gc/Transforms/IterativeTilingAndFusion.cpp

```diff
@@ -683,11 +705,13 @@ void iterativeTilingAndFusionUntilExhaustion(
         (void)mlir::simplifyRegions(rewriter, f->getRegions());
     } else {
       // Auto tiling with default tile size if no tiled op found. Follow tiling
-      // priority based on OpTy: `Contraction`->`Reduction`->`Elementwise`.
+      // priority based on OpTy:
+      // `ContractionOp`->`ReductionOp`->`LinalgOp`->`TensorOp`.
       SmallVector<DefaultTilingFn> priorityTilingPipeLine = {
           defaultTilingOfType<mlir::linalg::ContractionOpInterface>,
           defaultTilingOfType<mlir::linalg::ReduceOp>,
-          defaultTilingOfType<TilingInterface>};
+          defaultTilingOfType<mlir::linalg::LinalgOp>,
+          defaultTilingOfType<tensor::PackOp, tensor::UnPackOp, tensor::PadOp>};
```

Comment:
It's better to preserve the `tilingInterface` as a backup option here in case that other ops will implement `TilingInterface` in the future

Reply:
Ok, good suggestion! Add back now.